bot/src/main/java/com/almightyalpaca/discord/jdabutler/commands/commands/DateVersionCommand.java

```diff
@@ -66,6 +65,9 @@ public void dispatch(final User sender, final TextChannel channel, final Message
         }
         catch (IOException | NumberFormatException ex)
         {
+            final String fullStackTrace = getFullStackTrace(ex);
+            Bot.LOG.error("Exception in DateVersionCommand occured!" + System.lineSeparator() + fullStackTrace);
```

Comment:
Logging errors in slf4j works differently: `Logger#error(String message, Throwable throwable)`. You don't need to evaluate the stacktrace nor concatenate anything yourself.

Reply:
Didn't know there was something for this so I made it myself. Actually nice to see that this function seems to do the exact same as my function did. :D